e2e/interchaintestv8/testvalues/values.go

```diff
@@ -15,7 +15,7 @@ const (
 	StartingTokenAmount int64 = 10_000_000_000
 
 	// TransferAmount is the default amount of tokens to transfer between users.
-	TransferAmount int64 = 1_000
+	TransferAmount int64 = 1_000_000
```

Comment:
Should all the Amount types be uint instead of int?

Reply:
int64 this is easier to work with external libraries and auto generated code